src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -63,12 +64,14 @@ public RepoConfiguration(RepoLocation location, String branch, List<Format> form
 
         String organization = location.getOrganization();
         String repoName = location.getRepoName();
+        displayName = repoName + "[" + branch + "]";
+        outputFolderName = repoName + "_" + branch;
 
         if (organization != null) {
-            displayName = organization + "_" + repoName + "_" + branch;
             repoFolderName = organization + "_" + repoName;
+            displayName = organization + "/" + displayName;
+            outputFolderName = organization + "_" + outputFolderName;
         } else {
-            displayName = repoName + "_" + branch;
             repoFolderName = repoName;
```

Comment:
Move out of the `if-else` block, as suggested previously.

It's okay to remove the `final` keyword on `repoFolderName`.

Reply:
Ok.